domain-models.md

```diff
@@ -0,0 +1,107 @@
+# Domain Models and User Stories
```

Comment:
More of a personal preference thing, but I'd finish testing for one domain model before creating the next to help catch problems earlier on